hepmc2-macros/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "hepmc2-macros"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+proc-macro = true
+
+[dependencies]
+quote = "1.0.35"
+syn = { version = "2.0.48", features = ["full"] }
```

Comment:
Similar to the dependencies in the main crate I'd prefer to not pin the patch version.

Reply:
Done